bin/commands/init/index.sh

```diff
@@ -57,7 +57,7 @@ fi
 # zowe.runtimeDirectory
```

Comment:
This is probably overthinking, but:
* I don't have `node` installed (other user with sufficient permissions will do it)
* I would like install and config full Zowe (Desktop included)
* I can't run `zwe init` as the node is required due to app-server is enabled